packages/playwright/README.md

```diff
@@ -0,0 +1,58 @@
+# @knapsack-pro/playwright
+
+<p align="center">
+  <a href="https://knapsackpro.com?utm_source=github&utm_medium=readme&utm_campaign=knapsack-pro-playwright&utm_content=hero_logo">
+    <img alt="Knapsack Pro" src="./.github/assets/knapsack.png" width="100%" style="max-width: 100%;" />
```

Comment:
question

I'm not sure why, but for some reason, readme shows an image of Knapsack + Vitest instead of just Knapsack, despite the fact you added the proper image in this PR :/

https://github.com/KnapsackPro/knapsack-pro-js/tree/playwright/packages/playwright#knapsack-proplaywright
<img width="1109" height="611" alt="image" src="https://github.com/user-attachments/assets/91500076-85ca-41c6-b0bf-1dcc0734d250" />